test/e2e-tests-withdraw.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
+set -e
+
+cd dex-contracts/
+
+truffle exec scripts/request_withdraw.js 3 3 18
+truffle exec scripts/mine_blocks.js 21
+
+#TODO @josojo: this should happen by the driver
+truffle exec scripts/apply_withdrawals.js 0 0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 0x0000000000000000000000000000000000000000000000000000000000000001 0x0000000000000000000000000000000000000000000000000000000000000001
```

Comment:
I have already removed, this bitmap from the contracts. This is no longer necessary.

Reply:
Will have to pull the new contracts before we can remove this